src/Container/AddWmsPanel/AddWmsLayerEntry/AddWmsLayerEntry.spec.tsx

```diff
@@ -33,38 +35,39 @@ describe('<AddWmsLayerEntry />', () => {
   it('adds queryable icon if prop wmsLayer has queryable set to true', () => {
     testLayer.set('queryable', true);
 
-    render(<AddWmsLayerEntry wmsLayer={testLayer} />);
-    const icon = screen.getByLabelText('queryable-info');
-
-    expect(icon).toBeInTheDocument();
-    expect(icon).toHaveClass('fa-info');
+    // cannot seem to find the SVG element generated by FontAwesome using Testing Library.
+    // Doing it with Enzyme methods instead
```

Comment:
As `enzyme` is going to be removed in one of the next releases (hopefully), we should try get this working with rtl.

Reply:
I would favor a solution inside this PR.

Reply:
Managed to find  a solution. Problem was that the font awesome config needed to be imported to the test as well :sweat:

Reply:
You can check again